app/helpers/documents_v2_helper.rb

```diff
@@ -1,2 +1,15 @@
 module DocumentsV2Helper
+  def codap_v2_link(codap_server, runAsGuest=false, fromLaunchPage=false)
```

Comment:
You should be able drop the runAsGuest. That was added to support anonymous lara users opening codap documents.  Since all access through the v2 api is anonymous then we shouldn't need the runAsGuest param. @kswenson, perhaps there is something in the CFM or CODAP that still requires the runAsGuest param even with the lara provider?